tcf_website/templates/course/course.html

```diff
@@ -17,7 +17,7 @@
             {% include "../common/leaderboard_ad.html" with ad_slot="9533576562" %}
         </div>
         <div class="container-lg">
-            {% include "common/toolbar.html" with breadcrumbs=breadcrumbs active_text=latest_semester hidden_text="All" %}
+            {% include "common/toolbar.html" with breadcrumbs=breadcrumbs active_text=latest_semester hidden_text="All" sorting=True page="courses" %}
```

Comment:
I think instead of using a context variable to track what page you're on, you can just get the URL directly. See [here](https://stackoverflow.com/questions/2882490/how-to-get-the-current-url-within-a-django-template) for details.

Reply:
That makes sense- I updated `toolbar.html`, `department.html`, and `course.html` to reflect this change.